### ISSUE TITLE
rhyoids senpai

### ISSUE THREAD

**Issue by yurixoxo** [rhoiyds/osrs-streamers#11]
please update this beautiful plugin thank you

**Comment by royporter7** [comment]
What would you have updated?

**Comment by yurixoxo** [comment]
uhm.. well i am think its not working last time but let me try again

**Comment by royporter7** [comment]
Yes it’s currently not working. I have a big fix pending review from the
Runelite developers. It should be fixed in a few hours after approval.
Thank you for letting me know

**Comment by yurixoxo** [comment]
oh thank you so much! met lots of cool underground streamers from this so its really underrated imo

**Comment by yurixoxo** [comment]
> 
> 
> oh thank you so much! met lots of cool underground streamers from this so its really underrated imo

oh i meant to ask if the files are ok to be compiled already or like do you need to wait for runelite to give the ok

**Comment by royporter7** [comment]
Hello! I can confirm that just now the developers approved the bug fix, so
the plugin should now be functional again :)

**Comment by yurixoxo** [comment]
woo! thank you <3
